runtime/vm/rasdump.c

```diff
@@ -587,8 +587,13 @@ populateRASNetData(J9JavaVM *javaVM, J9RAS *rasStruct)
 	/* ensure that the string is properly terminated */
 	rasStruct->hostname[sizeof(rasStruct->hostname)-1] = '\0';
 
+#if defined(J9OS_I5_V7R3)
+	/* set AI_ADDRCONFIG(0x08) to fix low performance of inactive ipv6 resolving */
```

Comment:
Is this applicable to other platforms as well?  If there's a general improvement to made here, I'd like to see all platforms benefit.

Reply:
It does seem applicable. The man page mentions that if hints is NULL, then AI_ADDRCONFIG is the default (as well as AI_V4MAPPED).

Why is 0x08 used instead of AI_ADDRCONFIG?

Reply:
This issue is from a customer on ibm i. A simple java QIBMHello takes 5 seconds to initiate the JVM due to ipv6 query lookup. We do not find the issue on AIX based on customer's issue description. 
IBM i is different from AIX on processing the API getaddrinfo(). This issue seems to be ibm i specific including the flag value 0x08 which there is a flag conversion in ibm i slic codes.

Reply:
FYI.  Program "QIBMHello" is only a hello world test on i.

Reply:
@DanHeidinga  Yong Qing has answered your question.  Could you help to take a look?  Thanks.